### DISCUSSION TITLE
在进行表格识别时如何返回json格式的结果

### DISCUSSION THREAD

**Discussion by jweden33** [PaddlePaddle/PaddleOCR#13773] [Q&A]
返回的结果是excel格式文件和html格式的结果，我加了一个返回json格式的函数，但是识别的结果只能是逐行识别，不能每一个单元格的内容在一个字段中，如何将识别结果改为每个单元格的内容在一个字段中，求大佬指导
例："rec_res": [
            [
                "附件1",
                0.9971958994865417
            ],
            [
                "市分公司本部业务招待申请表",
                0.9953511357307434
            ],
            [
                "口市分公司领导",
                0.9886054992675781
            ],
            [
                "招待日期",
                0.9616472125053406
            ],
            [
                "招待类别",
                0.9926462769508362
            ],
            [
                "2023年2月201日",
                0.8390840291976929
            ],
            [
                "口部门领导",
                0.9716602563858032
            ],
            [
                "口商务",
                0.9536215662956238
            ],
            [
                "招待内容",
                0.9969135522842407
            ],
            [
                "招待标准",
                0.991277277469635
            ],
            [
                "区公务",
                0.8656947016716003
            ],
            [
                "弥迎奶商映肉队",
                0.4430771470069885
            ],
            [
                "来客单位",
                0.9962043762207031
            ],
            [
                "来客事由",
                0.9955308437347412
            ],
            [
                "治饺",
                0.647465705871582
            ],
            [
                "相",
                0.3445703089237213
            ],
            [
                "来客单位人数",
                0.9262593388557434
            ],
            [
                "10",
                0.7342853546142578
            ],
            [
                "招待陪同人数",
                0.9968960881233215
            ],
            [
                "3",
                0.9926198124885559
            ],
            [
                "就餐地点",
                0.9736618399620056
            ],
            [
                "口市分公司食堂",
                0.9943825602531433
            ],
            [
                "外部餐厅尔海南尚上如海",
                0.7167065143585205
            ],
            [
                "鱼酒店",
                0.7141678333282471
            ],
            [
                "经办人及联系电话",
                0.962116003036499
            ],
            [
                "珠茶1376681495",
                0.7795369029045105
            ],
            [
                "招待部门",
                0.9779266119003296
            ],
            [
                "悦兰",
                0.43004849553108215
            ],
            [
                "主要负责人意见",
                0.8950204849243164
            ],
            [
                "办公室",
                0.9986560940742493
            ],
            [
                "主要负责人审核",
                0.9713096618652344
            ],
            [
                "市分公司",
                0.9713602066040039
            ],
            [
                "分管领导审批",
                0.9979035258293152
            ],
            [
                "泰",
                0.2535126209259033
            ]
        ],

**Comment by SWHL** [comment]
现有的文本检测模型训练时采用的就是逐行识别的。
提供思路：先检测表格cell,将同一个cell的文本合并即可

**Reply by jweden33** [reply]
请问具体怎么合并呢 这是我不懂的地方
